### PR TITLE
moderator label

### PR DESCRIPTION
**Pull request by faissaloux**
- Add moderator label next to moderator/admin when reply on a thread.

### CODE REVIEW

**Comment by faissaloux** [comment]
Screenshot from 2021-09-07 14-16-36
Screenshot from 2021-09-07 14-14-33

**Comment by joedixon** [comment]
Thanks for the PR @faissaloux 

Do you think we could make the label in the same style as we do for tags on articles and posts?

https://github.com/laravelio/laravel.io/blob/main/resources/views/components/tag.blade.php

I think this component would be useful in other places and not just for users, so perhaps we could have it as a root component called `<x-lio-tag>`

**Comment by faissaloux** [comment]
@joedixon What do you think!

Screenshot from 2021-09-20 13-52-39

**Comment by joedixon** [comment]
Looks much better. Can we perhaps try with a transparent background?

**Comment by faissaloux** [comment]
@joedixon 

Screenshot from 2021-09-22 10-55-33

File: app/Models/User.php [hunk 1]
```diff
@@ -23,6 +23,11 @@ final class User extends Authenticatable implements MustVerifyEmail
     const MODERATOR = 2;
     const ADMIN = 3;
 
+    const LABELS = [
```
Comment (joedixon) [inline app/Models/User.php:26]: Not sure we need this when we're rendering both as `moderator`. I think we can just use `moderator` directly in the template.

Comment (faissaloux) [reply app/Models/User.php:26]: I thought of scenario if we want to use other labels later, if not the case I'll use `moderator` directly in the template

**Review by joedixon — APPROVED** [review]
Looks great, thanks @faissaloux

**Comment by faissaloux** [comment]
Happy to help 😄

**Pull request merged**